docs/gettingstarted/modfiles.md

```diff
@@ -248,3 +248,39 @@ An entry in `neoforge.mods.toml` does not need a corresponding `@Mod` annotation
 [versioning]: versioning.md
 [enumextension]: ../advanced/extensibleenums.md
 [featureflags]: ../advanced/featureflags.md
+## Mod files tree
+```
+root
+ |-- .gradle
+ |-- gradle
+ |-- build
+ |-- run
+ |-- src
+ |    |-- main
+ |    |    |-- java
+ |    |    |    |-- <com/net>
+ |    |    |    |    |-- <username>
+ |    |    |    |    |    |-- <modid>
+ |    |    |    |    |    |    |-- <some packages and single files>
+ |    |    |    |    |    |    |-- <modname>.java
+ |    |    |    |    |    |    |-- config.java
+ |    |    |-- resources
+ |    |    |    |-- assets
+ |    |    |    |    |-- <modid>
+ |    |    |    |-- data
+ |    |    |    |    |-- minecraft
+ |    |    |    |    |-- <modid>
+ |    |-- test
+ |-- build.gradle
+ |-- gradle.properties
+ |-- gradlew   (file)
+ |-- gradlew.bat
+ |-- settings.gradle
+```
+The `gradle` and `.gradle` folders are Gradle source folders (of the ModDevGradle plugin) that you will not normally need to modify.
+
+The `run` folder contains all data generated by testing your mod (like the `save` folder with all the world data).
```

Comment:
This is poorly explained. You should equate it to the vanilla `.minecraft` folder for a client or where a server generates its files.